shared_model/validators/abstract_validator.hpp

```diff
@@ -7,6 +7,7 @@
 #define IROHA_ABSTRACT_VALIDATOR_HPP
 
 #include "validators/answer.hpp"
+#include "validators/validators_common.hpp"
```

Comment:
I see no uses of this include in this file. My vote is against indirect includes, but it is not so important.